docs/sql-ref-datetime-pattern.md

```diff
@@ -199,11 +231,15 @@ The count of pattern letters determines the format.
 
 - Number/Text: If the count of pattern letters is 3 or greater, use the Text rules above. Otherwise use the Number rules above.
 
-- Fraction: Outputs the micro-of-second field as a fraction-of-second. The micro-of-second value has six digits, thus the count of pattern letters is from 1 to 6. If it is less than 6, then the micro-of-second value is truncated, with only the most significant digits being output.
+- Fraction: Use `'S..S'` pattern for parsing and formatting fraction of second.
```

Comment:
`S..S` is a bit confusing. How about `Use one or more contiguous 'S' characters ...`